lib/ext/heroku/command/addons.rb

```diff
@@ -174,7 +174,14 @@ def create
 
       action("Creating #{addon['name'].downcase}") {}
       action("Adding #{addon['name'].downcase} to #{app}") {}
-      action("Setting #{addon['config_vars'].join(', ')} and restarting #{app}") do
+
+      if addon.has_key? 'config_vars' && addon['config_vars']
```

Comment:
These all lose a check for the value being present (which [it will be](https://github.com/heroku/api/blob/f53c73994b7a116c97ef9f82830e253662bccc28/lib/api/serializers/addon_resource_serializer.rb#L12-L18)) vs being empty.

`if !addon['config_vars'].empty?` should be sufficient, `if !addon.fetch('config_vars', []).empty?` if you really want to be defensive.

In any case, the `config_vars` portion of the *resource* representation shouldn't be used, as it only ever shows config vars on the billing app. It is circumstantially OK in this context because we're only creating the initial attachment (i.e. to the billing app), though.